app/containers/MessageBox/MessageBoxContent.android.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
```

Comment:
You should merge MessageBoxContent into one file with both iOS and Android.
Too many duplicated code.